csharp-refactoring/RefactoringCouplers/RefactoringCouplers/FeatureEnvy/Correct/Ticket.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace RefactoringCouplers.FeatureEnvy.Correct;
+public class Ticket
+{
+    public Ticket(bool available, int price)
+    {
+        Available = available;
+        Price = price;
+    }
+
+    private bool Available { get; set; }
+    private int Price { get; init; }
+
+    public bool IsAvailable()
+    {
+        return Available;
+    }
```

Comment:
Good point, but I changed the property to `{ get; private set; }` because it is possible to change it outside the constructor (but not outside the `Ticket` class) :)